Tests/SwiftSyntaxMacrosTest/MacroSystemTests.swift

```diff
@@ -721,51 +673,56 @@ public let testMacros: [String: Macro.Type] = [
 ]
 
 final class MacroSystemTests: XCTestCase {
+  private let indentationWidth: Trivia = .spaces(2)
```

Comment:
`static`?

Reply:
But then I need to access it with `Self.indentationWidth` which is a little to verbose for my taste.